sound.js

```diff
@@ -239,7 +276,16 @@ Sound.unregisterHeadsetPlugChangeListener = function() {
       this.headsetPluggedInSubscription = null;
     }
   }
-  
+
+}
+
+Sound.unregisterOnPlayListener = function() {
```

Comment:
- Apart from the comment above, it should have been `Sound.prototype.unregisterOnPlayListener()`. You missed `.prototype`.
- If you agree on the comment earlier, this unregister codes could be moved to `Sound.destroy()` method.